develop/integrations.rst

```diff
@@ -1,6 +1,183 @@
 External Integrations
 #####################
 
+.. _cdc:
+
+Change Data Capture (CDC)
+=========================
+
+.. note::
+
+  Logical decoding for distributed tables is a beta feature in Citus v11.3.
+
+Differences from single-node PostgreSQL
+---------------------------------------
+
+PostgreSQL provides `logical decoding
+<https://www.postgresql.org/docs/current/logicaldecoding.html>`_ to export
+events of interest from its write-ahead log (WAL) to other programs or
+databases. The WAL keeps track of all committed data transactions; it is the
+authority on all data changes happening on a PostgreSQL instance. By
+subscribing to a logical decoding slot, an application can get real-time push
+notification of the changes.
+
+Consuming change events from Citus rather than single-node PostgreSQL adds two
+groups of challenges:
+
+1. **Client subscription complexity.** Any logical decoding publication must be
+   created in duplicate across all nodes. An interested application must
+   subscribe to all of them. Furthermore, events within each publication are
+   guaranteed to be emitted in the order they happen in the database, but this
```

Comment:
in the same "database" to "node" since WAL is shared across all databases in the same node.